include/mfmg/dealii/amge_host.templates.hpp

```diff
@@ -215,6 +215,11 @@ void anasazi_compute_eigenvalues_and_eigenvectors(
     std::vector<std::complex<double>> &eigenvalues,
     std::vector<dealii::Vector<double>> &eigenvectors)
 {
+#ifdef ANASAZI_TEUCHOS_TIME_MONITOR
```

Comment:
Would that be defined by Trilinos?

Reply:
Is it something that we could/should treat at configuration time or are we just making sure we don't accidentally call that function at runtime?

Reply:
We should just configure `Trilinos` appropriately.

Reply:
```
//Build with Teuchos TimeMonitors enabled.
Anasazi_ENABLE_TEUCHOS_TIME_MONITOR:BOOL=OFF
```